Formula/terminalimageviewer.rb

```diff
@@ -0,0 +1,25 @@
+class Terminalimageviewer < Formula
+  desc "Small program to display images in a terminal using RGB ANSI codes and unicode block graphics characters"
+  homepage "https://github.com/stefanhaustein/TerminalImageViewer"
+  url "https://github.com/stefanhaustein/TerminalImageViewer/archive/v1.0.0.zip"
+  sha256 "ab44f7d7b26152590a929fec6586e33a3b47737426467fc02c5d5149593ba094"
+  head "https://github.com/stefanhaustein/TerminalImageViewer.git"
+  depends_on "imagemagick"
+  depends_on "gcc" => :build unless OS.mac?
```

Comment:
`uses_from_macos "llvm"` is probably better

Reply:
That'll make it download LLVM on Linux. It's probably best to remove the line altogether - GCC is used by default on Linux.

Reply:
Actually, I put the wrong line there. I meant a line in which only on macos it will install gcc as clang doesn't support filesystem.

Reply:
Just remove the `unless` completely.